modules/ob-uninstalling-the-builds-operator.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// builds/uninstalling-openshift-builds.adoc
+
+:_content-type: PROCEDURE
+[id='ob-uninstalling-the-builds-operator_{context}']
+= Uninstalling the {builds-title} Operator
+
+You can uninstall the {builds-title} Operator by using the *Administrator* perspective in the web console.
+
+
+.Procedure
+
+. From the *Operators* -> *OperatorHub* page, use the *Filter by keyword* box to search for the *{builds-title}* Operator.
+
+. Click the *{builds-title}* Operator tile. The Operator tile indicates that the Operator is installed.
+
+. In the *{builds-title}* Operator description page, click *Uninstall*.
```

Comment:
Can we add the `Additional resources` section at the end, similar to what we have for [Pipelines](https://docs.openshift.com/container-platform/4.13/cicd/pipelines/uninstalling-pipelines.html#op-uninstalling-the-pipelines-operator_uninstalling-pipelines) ?

> Additional resources
> You can learn more about uninstalling Operators on OpenShift Container Platform in the [deleting Operators from a cluster](https://docs.openshift.com/container-platform/4.13/operators/admin/olm-deleting-operators-from-cluster.html#olm-deleting-operators-from-a-cluster) section.